python/lsst/sims/photUtils/Sed.py

```diff
@@ -481,6 +481,58 @@ def setFlatSED(self, wavelen_min=None,
         self.name = name
         return
 
+    def read_close_SED(self, teff, feH, logg):
+        """
+        Check the cached Kuruz models and load the model closest to the input stellar parameters
+        """
+        global _global_lsst_sed_cache
+
+        # Load the cache if it hasn't been done
+        if _global_lsst_sed_cache is None:
+            cache_LSST_seds()
+        # Build an array with all the files in the cache
+        if not hasattr(self, 'param_combos'):
+            kurucz_files = [filename for filename
+                            in _global_lsst_sed_cache if ('kurucz' in filename) &
+                            ('_g' in os.path.basename(filename))]
+            self.param_combos = numpy.zeros(len(kurucz_files),
```

Comment:
Can we rename this `self.kurucz_param_combos`, in case we end up implementing something similar for the other SED types.

Actually: would you be opposed to just making this a utility function, rather than a member of `Sed`?  It feels to me like this is not fundamental functionality that all `Sed`s need access to.  It is just a special way of loading an `Sed`.  I acknowledge that this is mostly a personal aesthetic choice.